docs/operations/move.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "Move"
+linkTitle: "Move"
+---
+
+## Moving Ondat volume instances
+
+A volume instance (replica or master) can be moved freely between nodes, this can be extremely useful in very specific scenarios, and it's available through a new command in the command-line tool.
+
+The process is safe to use and only ever removes data when a new instance of it has been synced within the new node.
+
+## Example of use
+
+Here's an example of what the CLI cmd expects:
+
+```bash
+storageos update volume move my-volume-id source-node-id destination-node-id --namespace my-namespace-name
+```
+
+## Moving to where an instance already exists
+
+When trying to move an Ondat volume instance into a node with another instance, the only scenario where this is valid is if the source node coincides with the master and the destination node with the replica, in which case we take an approach of moving the "roles" as opposed to the data itself thus attempt to promote the replica on destination node to be the new master instance of the Ondat volume.
```

Comment:
> When trying to move an Ondat volume instance into a node with another instance
This might be clearer if we specify it's the same volume, but a copy of the data i.e
> When trying to move an Ondat volume instance into a node with another instance (a replica with the same copy of the data)

Same point about "primary" vs "master"

Reply:
Fixed wording